tests/subsys/logging/log_api/src/test.inc

```diff
@@ -591,6 +591,13 @@ static void log_n_messages(uint32_t n_msg, uint32_t exp_dropped)
  */
 static void test_log_msg_dropped_notification(void)
 {
+	if (IS_ENABLED(CONFIG_SMP)) {
```

Comment:
Testing for `CONFIG_MP_NUM_CPUS > 1` might be better, since you can have only 1 CPU under SMP (this is usually used to test SMP code path).

Reply:
Alternatively use a _1cpu_ ztest test case here, which will lock down all but one CPU while the test runs (via a terrible hack, but it does work).  Definitely would be sad to lose test coverage on SMP systems.

Reply:
Gah, formating: `_1cpu_`

Reply:
@andyross it's using this new ztest API and i'm not sure what's the best way to run 1cpu test so i used @dcpleung suggestion. @yperess how to setup a single test case for 1 cpu?

Reply:
Yes, the best way is to create a new suite. The reason is that generally speaking you'll be adding more things to test additional code paths or edge cases to that suite. The thinking is to encourage small tests. In a way this is similar to adding `{}` to `if` statements even when there's just one statement. It's setting up for future changes.

One thing you could do if you wanted to avoid different test suites is to create a test rule that checks the test name. Something along the lines of if the test name ends with `_1cpu` then run the 1CPU start/stop functions.